### PR TITLE
[Enhance] Add abstract and a figure for each algorithm

### PR DESCRIPTION
**Pull request by Tai-Wang**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add abstract and a figure to provide a brief for each algorithm.

## Modification

Add abstract and a figure for each algorithm.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

No.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/free_anchor/README.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Modern CNN-based object detectors assign anchors for ground-truth objects under the restriction of object-anchor Intersection-over-Unit (IoU). In this study, we propose a learning-to-match approach to break IoU restriction, allowing objects to match anchors in a flexible manner. Our approach, referred to as FreeAnchor, updates hand-crafted anchor assignment to “free" anchor matching by formulating detector training as a maximum likelihood estimation (MLE) procedure. FreeAnchor targets at learning features which best explain a class of objects in terms of both classification and localization. FreeAnchor is implemented by optimizing detection customized likelihood and can be fused with CNN-based detectors in a plug-and-play manner. Experiments on COCO demonstrate that FreeAnchor consistently outperforms the counterparts with significant margins.
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/36950400/143866685-e3ac08bb-cd0c-4ada-ba8a-18e03cccdd0f.png" width="800"/>
```
Comment (Tai-Wang) [inline configs/free_anchor/README.md:12]: 800 is a little big (you can preview the image in the readme of the updated branch), maybe 600 is better?

File: configs/paconv/README.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+Furthermore, different from the existing point convolution operators whose network architectures are often heavily engineered, we integrate our PAConv into classical MLP-based point cloud pipelines without changing network configurations. Even built on simple networks, our method still approaches or even surpasses the state-of-the-art models, and significantly improves baseline performance on both classification and segmentation tasks, yet with decent efficiency. Thorough ablation studies and visualizations are provided to understand PAConv. Code is released on this https URL.
```
Comment (Tai-Wang) [inline configs/paconv/README.md:8]: "Code is released on xxx" can be deleted.

File: configs/parta2/README.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
...ect detection framework, the part-aware and aggregation neural network (Part-A2 net). The whole framework consists of the part-aware stage and the part-aggregation stage. Firstly, the part-aware stage for the first time fully utilizes free-of-charge part supervisions derived from 3D ground-truth boxes to simultaneously predict high quality 3D proposals and accurate intra-object part locations. The predicted intra-object part locations within the same proposal are grouped by our new-designed RoI-aware point cloud pooling module, which results in an effective representation to encode the geometry-specific features of each 3D proposal. Then the part-aggregation stage learns to re-score the box and refine the box location by exploring the spatial relationship of the pooled intra-object part locations. Extensive experiments are conducted to demonstrate the performance improvements from each component of our proposed framework. Our Part-A2 net outperforms all existing 3D detection methods and achieves new state-of-the-art on KITTI 3D object detection dataset by utilizing only the LiDAR point cloud data. Code is available at [this https URL](https://github.com/sshaoshuai/PointCloudDet3D).
```
Comment (Tai-Wang) [inline configs/parta2/README.md:7]: Code is xxx can be deleted

File: configs/regnet/README.md [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
 
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+In this work, we present a new network design paradigm. Our goal is to help advance the understanding of network design and discover design principles that generalize across settings. Instead of focusing on designing individual network instances, we design network design spaces that parametrize populations of networks. The overall process is analogous to classic manual design of networks, but elevated to the design space level. Using our methodology we explore the structure aspect of network design and arrive at a low-dimensional design space consisting of simple, regular networks that we call RegNet. The core insight of the RegNet parametrization is surprisingly simple: widths and depths of good networks can be explained by a quantized linear function. We analyze the RegNet design space and arrive at interesting findings that do not match the current practice of network design. The RegNet design space provides simple and fast networks that work well across a wide range of flop regimes. Under comparable training settings and flops, the RegNet models outperform the popular EfficientNet models while being up to 5x faster on GPUs.
```
Comment (Tai-Wang) [inline configs/regnet/README.md:7]: Add a figure

File: configs/ssn/README.md [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/79644370/143887479-8d493be7-8607-4bc2-acae-0d79ca0dd9e4.png" width="800"/>
```
Comment (Tai-Wang) [inline configs/ssn/README.md:12]: Replace it with the overall pipeline figure

**Comment by ZwwWayne** [comment]
@zhouzaida to take a look.

**Comment by zhouzaida** [comment]
LGTM. BTW, I will provide a script to verify whether those files `README.md` can be parsed successfully.

**Pull request closed without merging**